docs/csharp/tutorials/exploration/csharp-6.yml

```diff
@@ -64,7 +64,7 @@ items:
 - title: Importing a single class
   durationInMinutes: 2
   content: |
-    If you repeatedly use a class' static methods throughout your code, including the class name each time obscures the meaning of your code. The `Person` class currently imports the `System` namespace, even though only <xref:System.Console?displayProperty=nameWithType> is used. Modify the `using` statement as follows:
+    If you repeatedly use class' static methods throughout your code, including the class name each time obscures the meaning of your code. The `Person` class currently imports the `System` namespace, even though only <xref:System.Console?displayProperty=nameWithType> is used. Modify the `using` statement as follows:
```

Comment:
This one was correct. I wanted the possessive plural form of "class". Perhaps "one class' static methods" would be more clear?

Reply:
Ah! Got it :) yes, your change makes it clearer.